db/seeds.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+require './app/models/station.rb'
+
+def load_station(data)
+  csv_text = File.read(data)
+  csv = CSV.parse(csv_text, :headers => true)
+  csv.each do |row|
```

Comment:
I agree. Less steps is good, and I think ::foreach is more of a standard csv parse that is easier for others to recognize and understand quickly. Will update prior to merge.